### PR TITLE
Fix quit reason

### PR DESCRIPTION
**Pull request by rom1504**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ Server.prototype.listen = function(port, host) {
     var client = new Client(true);
     client._end = client.end;
     client.end = function end(endReason) {
+      endReason='{"text":"'+endReason+'"}';
```
Comment (roblabla) [inline src/server.js]: Wait so does this work in 1.8 ? Can we send an 1.8 client { "text": "Disconnected" } ?

Comment (rom1504) [inline src/server.js]: Yep, I just tried adding a 
```js

  setTimeout(function(){
    client.end("go away please");
  },15000);
```
at https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/examples/proxy/proxy.js#L84
and it just works :)

**Comment by roblabla** [comment]
Ready to merge ?

**Comment by rom1504** [comment]
yeah it's ready

**Pull request merged**